.github/CONTRIBUTING.md

```diff
@@ -30,52 +30,38 @@ Our website’s examples section is also our playground, please read the [Local
 
 ## Tests
 
+### Unit tests
+
 Unit tests are using Jest and can be run with:
 
 ```bash
 yarn run test:unit
 ```
 
-For end-to-end tests, we use [Webdriverio](http://webdriver.io). For it to run locally, you need to install a Selenium standalone server. Follow [the Webdriverio guide](https://webdriver.io/docs/selenium-standalone-service) to do so. You can also install a Selenium standalone server from NPM:
-
-```bash
-npm install selenium-standalone -g
-selenium-standalone install
-```
-
-And then launch it:
-
-```bash
-selenium-standalone start
-```
+### End-to-End tests
 
-After you have installed and launched the selenium standalone server, run:
+We use [Cypress](cypress.io/) for our e2e test suite. Be sure to checkout “[Writing your first test](https://docs.cypress.io/guides/getting-started/writing-your-first-test#Add-a-test-file)” and the “[Introduction to Cypress](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress#Cypress-Can-Be-Simple-Sometimes)”.
 
-```bash
-yarn run test:endtoend:local
-```
+<!-- TODO: check if this changed after writing -->
 
-By default, `test:endtoend:local` uses Firefox. You can use a different browser, like Chrome, by passing the `-b` flag:
+First, you need to create a `.env.local` in the root of Uppy. Depending on the tests, you will need this.
```

Comment:
I don't know if it's worth mentioning, but Vite supports several `.env` files so you could define a `.env.development` if there's a need to separate envs: https://vitejs.dev/guide/env-and-mode.html#env-files
Not sure it's worth putting in the docs, but wanted to share the info in case.

Reply:
Same for parcel. I forgot to update these docs but now did. We use `.env` now for everything.